src/main/java/net/sf/jabref/logic/util/strings/StringUtil.java

```diff
@@ -646,7 +646,14 @@ public static String expandAuthorInitials(String name) {
                 }
                 for (int j = 1; j < names.length; j++) {
                     if (j == 1) {
-                        sb.append(Author.addDotIfAbbreviation(names[j]));
+                        String[] splittedNames = names[j].split(" ");
+                        if (splittedNames.length > 0) {
+                            sb.append(Author.addDotIfAbbreviation(splittedNames[0]));
+                        }
+                        for (int k = 1; k < splittedNames.length; k++) {
+                            sb.append(' ');
+                            sb.append(Author.addDotIfAbbreviation(splittedNames[k]));
+                        }
```

Comment:
The problem here is that atleast to my experience, that a `StringJoiner` and `StringBuilder` or another `StringJoiner` has to be used. This is due to the reason that it first has to put the last name and first name together with a `", "` after that it should add that `"last name, first name"` with a `" and "` to the `String` that gets returned. I don't know if that increases simplicity. I suggest using one of koppor's propasals.

PS: Sorry for the stall, I had an operation this week and was resting.